lighthouse-core/closure/typedefs/viewer-externs.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/**
+ * Typing externs file needed for Viewer compilation.
+ * @externs
+ */
+
+/**
+ * @struct
+ * @constructor
+ */
+function ReportGenerator() {}
```

Comment:
rather than trying to compile `ReportGenerator` (with its `fs` use), just use externs for the one method we care about